composer.json

```diff
@@ -50,9 +50,26 @@
         }
     },
     "scripts": {
-        "lint": "@phpstan",
+        "lint": [
```

Comment:
would be nice to adjust this to match https://github.com/sulu/SuluFormBundle/pull/276

Reply:
would also install php-cs-fixer as dev-dependency and add a script for that